### ISSUE TITLE
left and right label for each node

### ISSUE THREAD

**Issue by aszharite** [rpocklin/angular-timeline#22]
Hello,

I was wondering if it is possible to display both labels on left and right side of a node at the same time?
This can be used when you have two storyboards and would like to show summary from both of them for each node.

Thanks

**Comment by rpocklin** [comment]
Not really, haven't had many people ask for this one.  How would you know when to show it on the same line?

**Comment by aszharite** [comment]
At the current iteration, you can specify a side on the timeline-event element. If you move that to the timeline-panel and have 2 panels one with side=right and one with side=left i think it would work

**Comment by rpocklin** [comment]
Happy to look at a PR.

**Comment by rpocklin** [comment]
Since there's been some interest, i've spiked an implementation on the branch `same-side-panel` - it's using `<timeline-row>` which would have two `<timeline-panel>...</timeline-panel><timeline-badge>...</timeline-badge>...<timeline-panel>...</timeline-panel>` inside it (spanning the whole row).

Be great to get feedback, here's a screenshot.
<img width="1363" alt="screen shot 2016-09-07 at 8 25 07 am" src="https://cloud.githubusercontent.com/assets/876651/18293239/e9e6d014-74d4-11e6-8cbe-30094fcf89b9.png">

Only current problem I can see with it is what happens < 768px - what do you think it should do?
